docker-compose.test.yml

```diff
@@ -32,6 +32,9 @@ services:
       - AUTH_HASH_ITERATIONS=130000
       - AUTH_HASH_KEY_LENGTH=64
       - AUTH_HASH_SALT_SIZE=32
```

Comment:
Do we need these `AUTH_...` vars now that there are SSAS versions?

Reply:
We need to keep the existing code working whilst we work on our new version.